### PR TITLE
   Update bot report

### PR DESCRIPTION
**Pull request by metaflow**
- add links to join beta and report issue
- add link "not useful" to clang-tidy warning
- clang-tidy comment in report now tells how many inline comments were
  added

### CODE REVIEW

File: docs/clang_tidy.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Not useful
```
Comment (ChristianKuehnel) [inline docs/clang_tidy.md]: Right now I'm pointing everyong at `user_doc.md` to understand what we're doing and how to use it.
Should we integrate this text there?  Or link it from there?

File: scripts/phabtalk/phabtalk.py [hunk 2]
```diff
@@ -65,7 +66,7 @@ def __init__(self, token: Optional[str], host: Optional[str], dryrun: bool):
     def dryrun(self):
         return self._phab is None
 
-    def _get_revision_id(self, diff: str):
+    def get_revision_id(self, diff: str):
```
Comment (ChristianKuehnel) [inline scripts/phabtalk/phabtalk.py]: just out of curiosity: Why did you rename the function?

Comment (metaflow) [reply scripts/phabtalk/phabtalk.py]: to use in  `title = 'Issue with build for {} ({})'.format(p.get_revision_id(args.diff_id), args.diff_id)` 
it was considered "private" with underscore

File: docs/clang_tidy.md (hunk 1 above)
Comment (metaflow) [reply docs/clang_tidy.md]: I have added link to and from user doc.
